docs/Web Scraper Cloud/API/Webhooks.md

```diff
@@ -18,7 +18,7 @@ A fresh webhook notification for the same scraping job can be sent:
 * When data extraction from empty and failed urls has been rescheduled. ("Continue" button in website)
 
 Notification FORM data content example:
-```
+```sh
```

Comment:
Let's keep in mind that this will also change styling in landing, we can adjust that with css on landing repo.

Without sh:
<img width="1332" alt="image" src="https://github.com/user-attachments/assets/cc2ed2f3-08b7-49cb-b39e-1c9172d5649d">

With sh:
<img width="1389" alt="image" src="https://github.com/user-attachments/assets/471a209e-5725-4727-84a8-da90e3b9408b">